samples/samples/noxfile.py

```diff
@@ -180,6 +180,7 @@ def blacken(session: nox.sessions.Session) -> None:
 # format = isort + black
 #
 
+
```

Comment:
Noxfile should not be updated manually (or by linters), because it's autogenerated. There is a note at the top of the file. The only exception is noxfile in `sqlalchemy_spanner`, because we written it by ourselves to create very special test sessions.